lib/modules/platform/azure/readme.md

```diff
@@ -91,6 +98,42 @@ module.exports = {
 
 For the `repositories` key, replace `YOUR-PROJECT/YOUR-REPO` with your Azure DevOps project and repository.
 
+### Using Azure DevOps internal API for pipeline tasks versions
+
+<!-- prettier-ignore -->
+!!! info
+    Renovate now uses the set of APIs that Azure provides to query the azure-pipelines tasks versions directly from the instance. Read [pull request #32966](https://github.com/renovatebot/renovate/pull/32966) and [discussion #24820](https://github.com/renovatebot/renovate/discussions/24820) for more background information on this change.
```

Comment:
```suggestion
    Renovate now uses the set of APIs that Azure provides to query the azure-pipelines tasks versions directly from the instance. Read [pull request 32966](https://github.com/renovatebot/renovate/pull/32966) and [discussion 24820](https://github.com/renovatebot/renovate/discussions/24820) for more background information on this change.
```

The `#` characters are commenting out the line. Let's remove the `#` bits. This way we can see the full sentence when published. 😉